components/SectionContainer/stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { storiesOf } from '@kadira/storybook'
+// import { WithNotes } from '@kadira/storybook-addon-notes'
```

Comment:
@andandandandrew did you want to add this back now that we got the dependency resolved?